fiboa_cli/convert.py

```diff
@@ -53,7 +57,10 @@ def list_all_converters(keys):
         try:
             converter = read_converter(id)
             for key in keys:
-                value = getattr(converter, key, None)
+                if isinstance(converter, BaseConverter):
+                    value = getattr(converter, key.lower())
+                else:
+                    value = getattr(converter, key, None)
```

Comment:
I think we should add comments to the parts that can be refactored/simplified once we reimplemented all converters to the class based approach.

```suggestion
                # todo: remove conditional once class-based converters have been fully implemented
                if isinstance(converter, BaseConverter):
                    value = getattr(converter, key.lower())
                else:
                    value = getattr(converter, key, None)
```

Reply:
Well, there's only a few places. I kept the changes as small as possible so we can do a staged refactoring.

Reply:
Yeah, but it would ensure that we remove everything once we actually have all of them converted to class based. Otherwise it could be very fiddly work to figure out what's still needed and what can be removed, I think.

Reply:
Ah, you want to add comments. Yes, makes sense. Is a `todo` like this good enough? `# todo: remove conditional once class-based converters have been fully implemented`

Reply:
See https://github.com/fiboa/cli/pull/138/commits/892c9a4a307ea082c979dd73d7ac771532495b1c

Reply:
Yes, that's excellent.